electron-builder.json

```diff
@@ -50,17 +50,14 @@
   "mac": {
     "category": "public.app-category.productivity",
     "target": [
-      "tar.gz"
-    ],
-    "extraResources": [
```

Comment:
The latest `electron-builder.json` should have proper config for extra resources. So you would not have to remove this config.